### DISCUSSION TITLE
Which terraform version is supported for terraform-azurerm-caf

### DISCUSSION THREAD

**Discussion by sunilnagavelli** [aztfmod/terraform-azurerm-caf#1844] [Q&A]
Hi Everyone, 

I have a quick question regarding the terraform version which can support this code, the reason for asking is, when i am trying to use this `appservice module`, i am seeing a complaint about syntax in dynamic block, The code is unaltered, and out of the box i have seen this error, could you please help me understand.

My Terraform Version: 
```
$ terraform version
Terraform v0.15.0
```
![image](https://github.com/aztfmod/terraform-azurerm-caf/assets/40353788/2459a01c-ac94-4405-b9cb-ea49f314171b)

**Comment by isevenbe** [comment]
The current version of the super module is working with terraform version 1.5.4
